pkg/apis/spinnaker/v1alpha2/spinnakeraccount_types.go

```diff
@@ -0,0 +1,85 @@
+package v1alpha2
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+type AccountType string
+
+const (
+	KubernetesAccountType AccountType = "Kubernetes"
+	AWSAccountType                    = "AWS"
+)
+
+type Authorization string
+
+const (
+	Read    Authorization = "READ"
+	Write                 = "WRITE"
+	Execute               = "EXECUTE"
+)
+
+type AccountPermissions map[Authorization][]string
+
+type SpinnakerAccountInterface interface {
+	GetAuth() map[string]interface{}
+	GetEnv() map[string]interface{}
+	GetSettings() map[string]interface{}
+	GetPermissions() *AccountPermissions
+}
+
+// SpinnakerAccountSpec defines the desired state of SpinnakerAccount
+// +k8s:openapi-gen=true
+type SpinnakerAccountSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
```

Comment:
This should be `make k8s openapi` right?

Reply:
That's left over from the code generation.